server/utils/mail_config.py

```diff
@@ -0,0 +1,72 @@
+import smtplib
+import logging
+from email.mime.text import MIMEText
+from email.mime.multipart import MIMEMultipart
+import os
+
+from dotenv import load_dotenv
+
+# Configure logging
+logging.basicConfig(
+    level=logging.DEBUG,
+    format="%(asctime)s [%(levelname)s] %(message)s"
+)
+logger = logging.getLogger(__name__)
+load_dotenv()
+# Loading configs from environment variables
+SMTP_SERVER = os.getenv("SMTP_SERVER")
+SMTP_PORT = int(os.getenv("SMTP_PORT", 465))
+SMTP_USER = os.getenv("SMTP_USER")
+SMTP_PASS = os.getenv("SMTP_PASS")
+ADMIN_EMAIL = os.getenv("ADMIN_EMAIL")
+SMTP_REPLY_EMAIL = os.getenv("SMTP_REPLY_EMAIL")
```

Comment:
Here, naming convention of flask env variables will also need to unify the naming.